.changeset/healthy-bugs-reply.md

```diff
@@ -0,0 +1,11 @@
+---
+"wrangler": patch
+---
+
+feat: dev+envs
+
+This implements service environments + `wrangler dev`. Fairly simple, it just needed the right url when creating the edge preview token.
+
+I tested this by publishing a service under one env, adding secrets under it in the dashboard, and then trying to dev under another env, and verifying that the secrets didn't leak.
+
+(This uncovered an issue where we can't set the right host incl. the env prefix, but it's not critical right now, following up internally.)
```

Comment:
This is rather enigmatic...

I don't think it is useful to have this here, unless there is something actionable by the person reading the changelog.

Reply:
Agreed, removed.